modules/features/cu_wysiwyg/cu_wysiwyg.features.filter.inc

```diff
@@ -255,7 +298,9 @@ sup,',
         'weight' => 50,
         'status' => 1,
         'settings' => array(
-          'local_paths' => _create_cu_pathologic_paths(),
+          'local_paths' => '/p136f5e238fe
```

Comment:
This line https://github.com/CuBoulder/express/pull/683/files/adc4aea5e1d40f58364c00deff1cb6225a4d7e90#diff-4d6190f2de94b8925de4e6211ffdcdd2R301